### PR TITLE
docs: video listing page videos added

### PR DESCRIPTION
**Pull request by rahulyadav139**
Demo: https://statuesque-creponne-891db8.netlify.app/

### CODE REVIEW

File: src/pages/single-video/SingleVideoPage.js [hunk 1]
```diff
@@ -49,30 +79,33 @@ const SingleVideoPage = props => {
...
-            <div className="video-comments__buttons">
-              <button>Cancel</button>
-              <button>Comment</button>
-            </div>
+            <input
+              onFocus={focusCommentHandler}
+              placeholder="Comment"
+              type="text"
+            />
+            {enableComment && (
```
Comment (j24m) [inline src/pages/single-video/SingleVideoPage.js:90]: Suggestion: if "enableComment" is a boolean then write prefix "is". This type of naming helps others to understand code better.

**Review by j24m — APPROVED** [review]

File: src/pages/single-video/SingleVideoPage.js [hunk 2]
```diff
@@ -1,37 +1,67 @@
...
+import { useFetch } from '../../hooks';
+import { useParams } from 'react-router-dom';
+import { dateFormatter } from '../../utils';
 
 const SingleVideoPage = props => {
+  const [videos, setVideos] = useState([]);
+  const [currentVideo, setCurrentVideo] = useState('');
+  const [enableComment, setEnableComment] = useState(false);
+  const { getData } = useFetch();
+  const params = useParams();
```
Comment (j24m) [inline src/pages/single-video/SingleVideoPage.js:13]: Suggestion: use object destructing instead of it.

File: src/utils/formatter.js [hunk 3]
```diff
@@ -7,4 +7,14 @@ const textFormatter = str => {
 
   return formattedStr;
 };
-export { textFormatter };
+
+const dateFormatter = date => {
+  const formattedDate = new Date(date).toLocaleString('en-IN', {
+    day: '2-digit',
```
Comment (mehulsatardekar) [inline src/utils/formatter.js:13]: really like this formatter function which basically transforme the numers with fixed 2 decimal and add prefix of indian currency symbol

**Review by mehulsatardekar — APPROVED** [review]

File: src/utils/formatter.js (hunk 3 above)
Comment (rahulyadav139) [reply src/utils/formatter.js:13]: Thank you so much.

**Pull request merged**